dwave_networkx/algorithms/matching.py

```diff
@@ -62,44 +219,22 @@ def maximal_matching(G, sampler=None, **sampler_args):
     References
     ----------
 
-    `Matching on Wikipedia <https://en.wikipedia.org/wiki/Matching_(graph_theory)>`_
+    `Matching on Wikipedia <https://w.wiki/r9s>`_
```

Comment:
Why use the short form here we space is not an issue and the longer form is more readable?

Reply:
To keep it under 80 characters